src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -49,7 +50,11 @@ def download_coverage_artifacts(build_task_id):
 
 def get_github_commit(mercurial_commit):
     url = 'https://api.pub.build.mozilla.org/mapper/gecko-dev/rev/hg/%s'
-    r = requests.get(url % mercurial_commit)
+    while True:
```

Comment:
yes, it can be safe to assume github mirror is 30min (or more) behind. sometimes it might even be a day, when conflicts happen and humans need to interact, rarely but still happens from time to time.

if possible i would avoid using github mirror, but I assume both coveralls/codecov don't work with mercurial.